plugins/pulp_rpm/plugins/importers/yum/repomd/presto.py

```diff
@@ -36,7 +37,7 @@ def process_package_element(element):
         oldepoch=delta.attrib['oldepoch'],
         oldversion=delta.attrib['oldversion'],
         oldrelease=delta.attrib['oldrelease'],
-        filename=filename.text,
+        filename=os.path.join("drpms", os.path.basename(filename.text)),
```

Comment:
Could you elaborate why this change is required?
We decided not to do anything with DRPMs and filename seems to include drpms dir already, iirc